imports/plugins/included/social/client/components/facebook.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from "prop-types";
 import Helmet from "react-helmet";
 import classnames from "classnames";
 import { $ } from "meteor/jquery";
-import { Translation } from "/imports/plugins/core/ui/client/components";
+import { Components } from "/imports/plugins/core/components/lib";
```

Comment:
Should we import instead from `import { Components } from "@reactioncommerce/reaction-components";`?